src/js/popup.js

```diff
@@ -91,6 +132,14 @@ async function getRemainingAliases() {
 }
 
 
+async function getDashboardData() {
+  const { aliasesUsedVal } = await browser.storage.local.get("aliasesUsedVal");
+  const { emailsForwardedVal } = await browser.storage.local.get("emailsForwardedVal");
+  const { emailsBlockedVal } = await browser.storage.local.get("emailsBlockedVal");
```

Comment:
Destructuring assignment again?
```suggestion
  const { aliasesUsedVal, emailsForwardedVal, emailsBlockedVal } = await browser.storage.local.get();
```